server-adobe@2018.json

```diff
@@ -27,5 +27,8 @@
             "disabled":false,
             "image":"modules_app/contentbox-custom/_content/ContentBoxIcon300.png"
         }
-    ]
+    ],
```

Comment:
Technically, since the webroot is the same directory, this is by convention. But I am ok with it being explicit

Reply:
I did not know that. I will make a few attempts with this option disabled and see if the logs show cfconfig is working.

Reply:
Luis, you are correct once again! So should I revert this commit?

![Screenshot from 2023-02-16 08-31-15](https://user-images.githubusercontent.com/8106227/219378369-48e769e9-5cce-456c-8b32-50c78fb10b0b.png)

Reply:
Go ahead and remove all of the `cfconfig` directives in the server.json files as they are the convention path